cves/kernel/CVE-2020-36558.yml

```diff
@@ -55,7 +55,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This vulnerability involves a potential crash when resizing a video terminal. 
+  The function uses a critical resource which, when accessed simultaneously (a.k.a Race Condition),
+  can result in a NULL pointer dereference, and a "General protection fault" error.
```

Comment:
I would expand on this. Provide some more context. Maybe get a little more detailed,

Reply:
Fixed